x-pack/test/security_solution_cypress/cypress/tasks/common.ts

```diff
@@ -92,3 +93,10 @@ export const waitForTabToBeLoaded = (tabId: string) => {
     }
   );
 };
+
+export const waitForPageToBeLoaded = () => {
+  cy.get(PAGE_CONTENT_SPINNER).should('be.visible');
+  cy.get(PAGE_CONTENT_SPINNER).should('not.exist');
+  cy.get(LOADING_INDICATOR).should('be.visible');
+  cy.get(LOADING_INDICATOR).should('not.exist');
```

Comment:
I think the flakiness reported by the "flaky test runner" comes from the order of these checks. In some scenarios the `LOADING_INDICATOR` may already be gone when Cypress realizes the `PAGE_CONTENT_SPINNER` does not exist. I suggest trying to check for the loaders are visible first, and then waiting until both of them are gone:

```suggestion
  cy.get(PAGE_CONTENT_SPINNER).should('be.visible');
  cy.get(LOADING_INDICATOR).should('be.visible');
  cy.get(PAGE_CONTENT_SPINNER).should('not.exist');
  cy.get(LOADING_INDICATOR).should('not.exist');
```